operator/src/main/java/com/github/streamshub/console/ConsoleReconciler.java

```diff
@@ -133,12 +134,35 @@
                 @Annotation(name = "createdAt", value = "placeholder"),
                 @Annotation(name = "support", value = "Streamshub")
             }),
-        description = "StreamsHub console provides a user interface for managing and monitoring your streaming resources",
+        description = "The Streamshub Console provides a web-based user interface tool for monitoring Apache Kafka® instances within a Kubernetes based cluster.\n\n "
```

Comment:
This might be better as a text block since it's multiple lines. E.g.
```java
description = """
   The StreamsHub Console provides...
   """
```
You can omit the `\n` characters in that case because newlines are taken literally in the block. wdyt?

Reply:
I did this at first, but it resulted in black background and code-style-like looking font, this way it looks clean. But i don't like that its like this either. It makes the editing more complicated.